modules/core/shared/src/main/scala/ciris/ConfigValue.scala

```diff
@@ -64,10 +68,29 @@ import ciris.ConfigEntry.{Default, Failed, Loaded}
 sealed abstract class ConfigValue[+F[_], A] {
   private[this] final val self: ConfigValue[F, A] = this
 
+  final def asIso[B](implicit codec: ConfigCodec[A, B]): ConfigValue[F, B] =
```

Comment:
I'm thinking we should rename `ConfigValue#asIso` to a different name to avoid `as` -> `asIso` -> `as` migration for end users. Maybe we can rename the internal `to` and then use at as the new name for `asIso`? So end users only migrate once from `as` -> `to`.